ofrak_components/ofrak_components/ecc/reedsolomon.py

```diff
@@ -0,0 +1,40 @@
+import reedsolo as rs
+from ofrak_components.ecc.abstract import EccAlgorithm, EccError
+
+
+class ReedSolomon(EccAlgorithm):
+    RSC = rs.RSCodec
```

Comment:
Is it intentional to define this as a class attribute? It doesn't look like it's ever used as a class attribute.

```suggestion
```